vss-extension.json

```diff
@@ -117,42 +117,76 @@
                     "displayName": "Marketplace URL",
                     "value": "https://marketplace.visualstudio.com"
                 },
-                "authenticationSchemes": [
+              "authenticationSchemes": [
+                {
+                  "type": "ms.vss-endpoint.endpoint-auth-scheme-token",
```

Comment:
Generic would work, but is ugly. I think I broke compatibility... By trying
to be clever. I appended vsts to the connection and used the value of the
radio button to select the right connection. I can fix that.